apps/scan/backend/test/helpers/app_helpers.ts

```diff
@@ -317,3 +321,40 @@ export async function configureApp(
   await apiClient.setTestMode({ isTestMode: false });
   await apiClient.setPollsState({ pollsState: 'polls_open' });
 }
+
+/**
+ * Interpretation is generally the slowest part of tests in this file. To speed
+ * up a test, you can use this function to mock interpretation. It should only
+ * be used when:
+ * - The test isn't meant to check that interpretation works correctly. There
+ *   should already be another test that covers the same interpretation case.
+ * - The test doesn't check the CVR export at the end. The interpreter stores
+ *   the ballot images which are used in the CVR, and mocking will forgo that
+ *   logic.
+ * - The test doesn't depend on the actual page interpretations. This function
+ *   adds fake page interpretations that don't actually match the passed in
+ *   ballot interpretation (because the state machine doesn't actually use those
+ *   page interpretations, they are just stored for the CVR).
+ */
+export function mockInterpretation(
```

Comment:
nit: reminder to remove the original copy of this function - noticed it's still in `app_scan.test.ts`

Reply:
Thanks, did that in bb2166f as well as removing the duplicate `checkLogs` functions.